src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/StringParsingHelpers.Addresses.cs

```diff
@@ -11,14 +11,8 @@ internal static partial class StringParsingHelpers
     {
         // /proc/net/route contains some information about gateway addresses,
         // and separates the information about by each interface.
-        internal static List<GatewayIPAddressInformation> ParseGatewayAddressesFromRouteFile(string filePath, string interfaceName)
+        internal static List<GatewayIPAddressInformation> ParseIPv4GatewayAddressesFromRouteFile(List<GatewayIPAddressInformation> collection, string filePath, string interfaceName)
```

Comment:
Do we still need to return the collection, since it's being passed in?

Reply:
I was thinking about it but I was not sure what should we do? void?